Bots Explained

```diff
@@ -0,0 +1,34 @@
+When my Mentor Oscar Oranagwa asked me to write about bots, I decided to use this opportunity to clarify exactly on what bots
+are, to both my fellow developers as well as any newbie with the desire to benefit from this innovation. I will use a story
+to explain it all to an extent but before that, let me begin with;
+
+WHAT ARE BOTS?
+BOTS are also known as Internet Robots, and just as the name Robot implies; Bots make things easier, they can handle 
+repetitive tasks and many more. Formerly, the popular version of bots came in the form of malware, but that has begun 
```

Comment:
Can you please correct all my crazy line breaks... They happened because of how I wrote it